### PR TITLE
Test basic workflows

### PR DESCRIPTION
**Pull request by domoritz**
- [ ] Does this PR change the contracted DBG, catlas, or minhash DB formats?
      If so, merge with care as this will require rebuilding some big files.

### CODE REVIEW

**Comment by domoritz** [comment]
I need to run everything up to search because otherwise the log output doesn't just have the search results.

**Comment by ctb** [comment]
This is pretty distinct from what I implemented in #116 - I think both are valuable. This takes longer and shouldn't be run automatically; the one in #116 is useful for smoke testing and integration testing ("does it work at all", and "does it all work together").

File: spacegraphcats/rdomset.py [hunk 1]
```diff
@@ -2,7 +2,7 @@
 from collections import defaultdict
 from .graph import Graph, DictGraph
 
-from typing import List, Set, Dict, Any, Union
+from typing import List, Set, Dict, Any, Union, DefaultDict
```
Comment (ctb) [inline spacegraphcats/rdomset.py]: What version of Python are you using? This isn't in Python 3.5 or before.

Comment (ctb) [inline spacegraphcats/rdomset.py]: Dict works fine.

Comment (domoritz) [inline spacegraphcats/rdomset.py]: This is weird. I saw failures on travis because of this. On my machine, I'm using Python 3.6.1.

**Pull request merged**